src/Analyzer.TemplateProcessor/PlaceholderInputGenerator.cs

```diff
@@ -186,7 +186,8 @@ internal static InsensitiveDictionary<JToken> GeneratePlaceholderDeploymentMetad
                 { "subscription", new JObject(
                     new JProperty("id", "/subscriptions/00000000-0000-0000-0000-000000000000"),
                     new JProperty("subscriptionId", "00000000-0000-0000-0000-000000000000"),
-                    new JProperty("tenantId", "00000000-0000-0000-0000-000000000000")) },
+                    new JProperty("tenantId", "00000000-0000-0000-0000-000000000000"),
+                    new JProperty("displayName", "aSubscriptionName")) },
```

Comment:
nit: maybe just 'SubscriptionName'

Reply:
I would rather leave it as it is to make it more clear that is just one possible subscription name